src/shared/clients/ecsClient.ts

```diff
@@ -5,70 +5,69 @@
 
 import { ECS } from 'aws-sdk'
 import { ext } from '../extensionGlobals'
+import { getLogger } from '../logger'
 import { ClassToInterfaceType } from '../utilities/tsUtils'
 
 export type EcsClient = ClassToInterfaceType<DefaultEcsClient>
+
+export type ServicesAndToken =  {
+    services: ECS.Service[],
+    nextToken?: string
+}
+
+export type ClustersAndToken = {
+    clusters: ECS.Cluster[],
+    nextToken?: string
+}
+
+const MAX_RESULTS = 50
```

Comment:
Nit: could use a more descriptive name, maybe `MAX_CLUSTER_PAGE_SIZE` ?

Reply:
I need to go with a more general name since it's used  in more than type of sdk call.  Changed to `MAX_RESULTS_PER_RESPONSE`